src/Fields/interpolate.jl

```diff
@@ -252,7 +252,7 @@ left bound `i⁻`, such that `ξ ∈ [0, 1)`.
 
     i⁻ = Base.unsafe_trunc(Int, fractional_idx)
     i⁻ = Int(i⁻ + 1) # convert to "proper" integer?
-    i⁺ = i⁻ + 1
+    i⁺ = i⁻ + 1 * Int(sign(fractional_idx))
```

Comment:
```suggestion
    shift = Int(sign(fractional_idx))
    i⁺ = i⁻ + shift
```

Not sure I understand what this is doing, but I don't think we need to multiply anything by `1` right?

Reply:
yeah not sure why I wrote it that way before